source/m2mconnectionsecuritypimpl.cpp

```diff
@@ -393,8 +393,8 @@ int entropy_poll( void *, unsigned char *output, size_t len,
     } else {
         rdm = time(NULL);
     }
+    srand(rdm);
```

Comment:
This code will be removed anyway when we merge the PAL TLS version https://github.com/ARMmbed/mbed-client-mbed-tls/pull/63.